browser-es6.js

```diff
@@ -21,5 +21,12 @@ module.exports = merge(base, {
   },
   plugins: [
     'react'
-  ]
+  ],
+  rules: {
+    'new-cap': 0,
+    'no-extra-parens': 0,
+    'no-inline-comments': 0,
```

Comment:
Can you change this at the base level instead? Imo inline comments make code a lot more difficult to read but I can see the value, and the current setting kinda goes against the style in all our repos